homeassistant/components/zwave_js/discovery.py

```diff
@@ -72,11 +73,11 @@ class ZWaveDiscoverySchema:
     # [optional] the node's firmware_version must match ANY of these values
     firmware_version: Optional[Set[str]] = None
     # [optional] the node's basic device class must match ANY of these values
-    device_class_basic: Optional[Set[str]] = None
+    device_class_basic: Optional[Set[Union[str, int]]] = None
```

Comment:
How come it can be either a string or an int? Shouldn't we make it a single type. Does ZJS Server give us either?

Reply:
Oh I see for string we still compare to a label. I guess we plan to phase that out in a future PR?

Reply:
correct but it would cost me hours to find those int values with the strings